inst/include/Rcpp/sugar/functions/strings/trimws.h

```diff
@@ -133,7 +133,8 @@ inline Vector<STRSXP> trimws(const Vector<STRSXP>& x, const char* which = "both"
 }
 
 inline Matrix<STRSXP> trimws(const Matrix<STRSXP>& x, const char* which = "both") {
-    R_xlen_t i = 0, nr = x.nrow(), nc = x.ncol(), sz = x.size();
+    R_xlen_t i = 0, sz = x.size();
+    int nr = x.nrow(), nc = x.ncol();
```

Comment:
Me too, was just trying to confirm.  Because otherwise nrow, ncol should not be int here and in Matrix.  

Have a test running with the now-merged PR.  Things take some time of course.